frontend/components/Layout/Footer.jsx

```
@@ -55,70 +55,81 @@ const ContributorGroup = () => {
-                {c.linkedin && (
-                  <IconButton
-                    href={c.linkedin}
-                    target={"_blank"}
-                    onClick={() => {
-                      trackEvent(`button.${c.login}.linkedin.click`, {
-                        type: "footer",
-                      });
-                    }}
-                    as={"a"}
-                    size={"sm"}
-                    aria-label={"LinkedIn"}
-                    icon={<FaLinkedinIn size={20} />}
-                  />
-                )}
-                {c.website && (
-                  <IconButton
-                    href={c.website}
-                    target={"_blank"}
-                    onClick={() => {
-                      trackEvent(`button.${c.login}.website.click`, {
-                        type: "footer",
-                      });
-                    }}
-                    as={"a"}
-                    size={"sm"}
-                    aria-label={"Website"}
-                    icon={<FaHome size={20} />}
-                  />
-                )}
-                {c.github && (
-                  <IconButton
-                    href={c.github}
-                    target={"_blank"}
-                    onClick={() => {
-                      trackEvent(`button.${c.login}.github.click`, {
-                        type: "footer",
-                      });
-                    }}
-                    as={"a"}
-                    size={"sm"}
-                    aria-label={"Github"}
-                    icon={<FaGithub size={20} />}
-                  />
-                )}
-              </HStack>
-            </VStack>
-          </WrapItem>
-        ))}
+        {bigContributors.length === 0 && (
+          <Text
+            textAlign={"center"}
+            fontSize={20}
+            fontWeight={400}
+            color={"gray.600"}
+          >
+            Failed to load contributors
+          </Text>
+        )}
```

```suggestion
```

Let's prefer silently failing over showing text -- this is sometimes flashing to users who are loading the website for the first time.